providers/rxe/rxe.c

```diff
@@ -1528,7 +1528,7 @@ static int post_one_send(struct rxe_qp *qp, struct rxe_wq *sq,
 
 	err = validate_send_wr(qp, ibwr, length);
 	if (err) {
-		printf("validate send failed\n");
+		fprintf(stderr, "rxe: validate send failed\n");
```

Comment:
These prints or should be deleted to converted to general log infrastructure.

See efa as an example.